client/app/routes/market.tsx

```diff
@@ -20,12 +26,32 @@ export const loader = async ({ request }: LoaderFunctionArgs) => {
 
 	const sellerProfile = user
 		? await prisma.sellerProfile.findFirst({
-				where: { userId: user.id },
+				where: { userId: user?.id },
 			})
 		: null;
 
+	const searchParams = new URL(request.url).searchParams;
+	const { q, category } = qs.parse(searchParams.toString(), {
+		ignoreQueryPrefix: true,
+	});
+
+	const whereClause: any = {
+		status: "available",
+	};
+
+	if (q) {
+		whereClause.OR = [
+			{ name: { contains: q.toString(), lte: "insensitive" } },
```

Comment:
What does `lte: "insensitive` do?

Reply:
Case-sensitive matching in Prisma. "Mode: "insensitive"" was throwing an error. It's not even necessary; It works without it anyway. I'll remove that